perfact/zodbsync/commands/watch.py

```diff
@@ -359,11 +359,7 @@ def run(self, interval=10):
             raise
 
         # Log in as a manager
-        uf = self.app.acl_users
-        user = uf.getUserById(self.sync.manager_user)
-        if not hasattr(user, 'aq_base'):
-            user = user.__of__(uf)
-        AccessControl.SecurityManagement.newSecurityManager(None, user)
+        self.sync.start_transaction().commit()
```

Comment:
Ah, this is now in `zodbsync.py`! I see…

Reply:
Actually, while experimenting with #55, I was wondering why `create_manager_user` had no effect, until I saw that this code segment was duplicated in the watcher, but without handling the case that there might be no users.